src/client/dfs/duns.c

```
@@ -1247,7 +1247,9 @@ duns_link_cont(daos_handle_t poh, const char *cont, const char *path)
 #ifdef LUSTRE_INCLUDE
 		struct statfs   fs;
 		char            *dir, *dirp;
+		size_t          path_len;
 
+		path_len = strnlen(path, PATH_MAX);
```

if path's len exceeds PATH_MAX, path will be silently truncated, is that ok?